OWNERS

```diff
@@ -1,18 +1,10 @@
 approvers:
-  - animeshsingh
   - chensun
   - james-jwu
-  - knkski
-  - shannonbradshaw
+  - thesuperzapper
   - zijianjoy
 reviewers:
```

Comment:
We need a clear definition of this. I feel like this should be the same as approvers at root level since we already have repo/component level OWNERs.

Reply:
I am a little confused about what you are saying here.

This PR is specifically about the root OWNERS for the repo, which are a "fallback" if a more specific reviewer/approver is not defined for a PR.

The main issue is that the websites' structure almost always requires a root approver due to configs being at relatively top-level folders (e.g. redirects, and git-submodules for the docsy theme).